src/evict/evict_inline.h

```diff
@@ -616,15 +618,16 @@ __evict_check_user_ok_with_eviction(WT_SESSION_IMPL *session, bool busy)
  *     Input parameters:
  *       (1) `busy`: A flag indicating if eviction is mandatory (true) or optional (false).
  *       (2) `readonly`: A flag indicating if the session is read-only, in which case dirty and
- *            update triggers are ignored.
- *       (3) `didworkp`: A pointer to indicate whether eviction work was done (optional).
+ *          update triggers are ignored.
+ *       (3) `interruptible`: A flag indicating if eviction is ok to be interrupted by user.
```

Comment:
Wordsmithing suggestion:

"A flag indicating if the user is allowed to interrupt eviction"